### PR TITLE
Devx 1202 delete docs

### PR DESCRIPTION
**Pull request by karyamane**
Linked to https://izettle.atlassian.net/browse/DEVX-1202

### CODE REVIEW

**Review by Astro-Choco — APPROVED** [review]
Perfect

File: README.md [hunk 1]
```diff
@@ -1,496 +1,6 @@
- [![CocoaPods](https://img.shields.io/cocoapods/v/iZettleSDK.svg)](https://cocoapods.org/pods/iZettleSDK) [![CocoaPods](https://img.shields.io/cocoapods/p/iZettleSDK.svg)](https://cocoapods.org/pods/iZettleSDK)
-
-:warning: **Starting May 9, 2022, all the documentation for iOS SDK is available on [Zettle Developer Portal](https://developer.zettle.com).**
```
[the remark this answers is not in the document]
Comment (MBelogorodtsev-IZ) [reply README.md:3]: Yep, my mistake, didn't notice that you've already added this sentence to the end of this file :)

Approving the PR

[the remark this answers is not in the document]
Comment (karyamane) [reply README.md:3]: yes @MBelogorodtsev-IZ, as discussed, I have the READMe file even now with the message redirecting to developer portal.

**Review by MBelogorodtsev-IZ — APPROVED** [review]
LGTM 👍

**Pull request merged**